.yamato/config/universal_upgrade.metafile

```diff
@@ -0,0 +1,33 @@
+project:
+  name: urp-upgrade
+  folder: UniversalUpgradeTest
+test_platforms:
+  - type: editmode
+    commands:
+      - [all]: npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+      # - [all]: upm-ci package pack --package-path com.unity.testing.urp-upgrade
+      - [all]: upm-ci project pack --project-path TestProjects/UniversalUpgradeTest
```

Comment:
It might be worth moving this out to a separate pack pipeline so it can be re-used for the publish and promote jobs, but this should be fine for testing